### PR TITLE
add document management system tables

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

**Comment by 3nids** [comment]
Now with gen_random_uuid

**Comment by 3nids** [comment]
can we proceed with the merge?

**Comment by ponceta** [comment]
@3nids We miss the delta file, otherwise it looks good to me.

**Comment by 3nids** [comment]
@ponceta done, feel free to merge / ask someone to do so / give me permission to merge

File: init_qwat.sh [hunk 1]
```diff
@@ -126,6 +126,7 @@ fi
 # Create extenstions
 psql service=${PGSERVICE} -v ON_ERROR_STOP=1 -c "CREATE EXTENSION IF NOT EXISTS hstore;"
 psql service=${PGSERVICE} -v ON_ERROR_STOP=1 -c "CREATE EXTENSION IF NOT EXISTS postgis;"
+psql service=${PGSERVICE} -v ON_ERROR_STOP=1 -c 'CREATE EXTENSION IF NOT EXISTS "pgcrypto";'
```
Comment (ponceta) [inline init_qwat.sh:129]: @3nids This statement is missing in the delta file and will result in a sql failure when updating, could you please add it to the delta file?

**Comment by ponceta** [comment]
Once my last comment is solved, we can merge this.

I see that the unit tests do no longer work on qwat db model. 

I've opened an issue for that : https://github.com/qwat/qwat-data-model/issues/348

File: init_qwat.sh (hunk 1 above)
Comment (3nids) [reply init_qwat.sh:129]: done

**Pull request merged**